src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -527,15 +603,21 @@ def _set_gradient_checkpointing(self, module, value=False):
     Args:
         pixel_values (`torch.FloatTensor` of shape `(batch_size, num_channels, height, width)`):
             Pixel values.
-        input_ids (`torch.LongTensor` of shape `(batch_size * num_max_text_queries, sequence_length)`):
+        input_ids (`torch.LongTensor` of shape `(batch_size * num_max_text_queries, sequence_length)`, *optional*):
             Indices of input sequence tokens in the vocabulary. Indices can be obtained using [`CLIPTokenizer`]. See
             [`PreTrainedTokenizer.encode`] and [`PreTrainedTokenizer.__call__`] for details. [What are input
-            IDs?](../glossary#input-ids)
+            IDs?](../glossary#input-ids) Either pass `input_ids` or `query_pixel_values`.
+        query_pixel_values (`torch.FloatTensor` of shape `(batch_size, num_channels, height, width)`, *optional*):
```

Comment:
Just a side note here - if we include query_pixel_values as third element of the public forward API, then users might need to update their code as they currently assume attention_mask is the third element.

But given that this is a new model, I think it's fine to add this functionality (as image-guided object detection is one of the main contributions of OWL-ViT). cc @sgugger

Reply:
@NielsRogge good point, I'm working on rebasing this branch to main and aiming for as much as backward compatibility as possible.